tuxemon/event/conditions/common.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+import logging
+
+from tuxemon.npc import NPC
+
+logger = logging.getLogger(__name__)
+
+
+class CommonCondition:
+    name = "Common"
+
+    @staticmethod
+    def check_character_attribute(
+        character: NPC,
+        attribute: str,
+        value: str,
+    ) -> bool:
+        """
+        Check a character's (npc or player) attribute.
```

Comment:
I feel like this docstring is not explaining anything.

"Check a character's attribute", "The NPC object to check" etc.

But what does it mean "checking" in this context? It's a so generic term that it could be beneficial to elaborate a bit about what is done there.

Reply:
I agree, it's confusing